third_party/terraform/data_sources/data_source_google_sql_ca_certs.go

```diff
@@ -0,0 +1,108 @@
+package google
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceGoogleSQLCaCerts() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceGoogleSQLCaCertsRead,
+
+		Schema: map[string]*schema.Schema{
```

Comment:
nit: Can you order this such that the fields go in the order Required -> Optional -> Computed?

Reply:
Done